Service/IndexService.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace EMS\CoreBundle\Service;
+
+use Elasticsearch\Client;
+use Psr\Log\LoggerInterface;
+
+final class IndexService
+{
+    /** @var AliasService */
+    private $aliasService;
+    /** @var Client */
+    private $elasticSearchClient;
```

Comment:
I think we always use 'client' as variable name for this one.

Reply:
OK